services/worker-manager/test/worker_scanner_test.js

```diff
@@ -78,6 +81,8 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
         workerId: 'testing-123',
       });
       assert(worker.providerData.checked);
+      // expires wasn't updated
```

Comment:
maybe worth a comment to explain expires should only be updated if less than a day, and expires2 is not in 3 days